policyengine_us/variables/gov/states/az/tax/income/deductions/standard/az_standard_deduction.py

```diff
@@ -13,4 +13,16 @@ class az_standard_deduction(Variable):
     def formula(tax_unit, period, parameters):
         p = parameters(period).gov.states.az.tax.income.deductions.standard
         filing_status = tax_unit("filing_status", period)
-        return p.amount[filing_status]
+        # return p.amount[filing_status]
+
+        charitable_deduction = tax_unit("charitable_deduction", period)
```

Comment:
make the increased standard deduction its own variable and then just add it here, we want it to be calculated separately

Reply:
so one file az_increased_standard_deduction - just the charitable contributions part 

one file az_standard_deduction - the deduction as it was + az_increased_standard_deduction output